turbo.json

```diff
@@ -4,7 +4,7 @@
   "tasks": {
     "build": {
       "dependsOn": ["^build"],
-      "outputs": [".next/**", "!.next/cache/**"]
+      "outputs": [".next/**", "!.next/cache/**", "dist/**"]
```

Comment:
Why is this added? :)

Reply:
<img width="1251" alt="image" src="https://github.com/user-attachments/assets/0e9d6837-3a8d-444d-b00b-f826e30fe172" />
As I understand it, It's to cache built files. We already cached nextjs apps with `".next/**"` but we had no config for our packages that we publish to npmjs. This config should probably have been there from the beginning, but adding now as I think the above warning might be new.